packages/ui/src/components/va-button/VaButton.vue

```diff
@@ -251,185 +221,161 @@ export default defineComponent({
     }
   }
 
-  &--default {
-    color: var(--va-button-background-color, var(--va-white));
-
-    i {
-      color: var(--va-button-icon-color, var(--va-white));
-    }
-  }
+  &--small {
+    @include va-button(var(--va-button-sm-content-py), var(--va-button-sm-content-px), var(--va-button-sm-font-size), var(--va-button-sm-line-height), var(--va-button-sm-border-radius));
```

Comment:
Can you follow this up by issue (or point me to one if it's created already)? I think we reached an agreement that's not a satisfiable solution long term. Works for this pr though!

Reply:
Will fix it here.